modules/rest-http/src/main/java/org/apache/ignite/internal/processors/rest/protocols/http/jetty/GridJettyObjectMapper.java

```diff
@@ -154,6 +154,13 @@ private void writeException(Throwable e, JsonGenerator gen) throws IOException {
 
             writeException(e, gen);
 
+            gen.writeArrayFieldStart("stackTrace");
```

Comment:
This change should be extracted to separate issue, because it will be backported to 8.x-master branches.

Reply:
Created separated issue for stacktrace serialisation.